geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert/common/TransformersTest.scala

```diff
@@ -866,5 +866,106 @@ class TransformersTest extends Specification {
       trans.eval(Array("bar")) mustEqual "bar"
       trans.eval(Array("")) mustEqual ""
     }
+
+    "strip strings" >> {
+
+      "strip quotes" >> {
+        val trans = Transformers.parseTransform("stripQuotes($0)")
+        trans.eval(Array("'foo'")) mustEqual "foo"
+        trans.eval(Array("\"foo'")) mustEqual "foo"
+        trans.eval(Array("\"'foo'")) mustEqual "foo"
+
+        // white space is preserved
+        trans.eval(Array("'foo\t\t")) mustEqual "foo\t\t"
+        trans.eval(Array("  foo'\"\"")) mustEqual "  foo"
```

Comment:
If we spent more time on this, I think I'd use Scala's """ rather than escaping quotes:).